Tests/UpstreamIntegrationTests/UpstreamIntegrationTests.swift

```diff
@@ -0,0 +1,200 @@
+        MobileCore.setLogLevel(.trace)
+        // Set environment file ID for specific Edge Network environment
+        setMobileCoreEnvironmentFileID(for: edgeEnvironment)
+        MobileCore.registerExtensions([Identity.self, Edge.self], {
+            print("Extensions registration is complete")
+            waitForRegistration.countDown()
+        })
+        XCTAssertEqual(DispatchTimeoutResult.success, waitForRegistration.await(timeout: 2))
+        
+        // Set Edge location hint value if one is set for the test target
+        if edgeLocationHint != nil {
+            print("Setting Edge location hint to: \(String(describing: edgeLocationHint?.rawValue))")
+            Edge.setLocationHint(edgeLocationHint?.rawValue)
+        }
+        else {
+            print("No preset Edge location hint is being used for this test.")
+        }
+    }
+    
+    public override func tearDown() {
+        super.tearDown()
+
+        // to revisit when AMSDK-10169 is available
+        // wait .2 seconds in case there are unexpected events that were in the dispatch process during cleanup
+        usleep(200000)
+        EventHub.reset()
+        UserDefaults.clearAll()
+        FileManager.default.clearCache()
+    }
+    
+    // MARK: - Upstream integration test cases
+    
+    func testSendEvent_withStandardExperienceEvent_receivesExpectedEventHandles() {
+        // Setup
+        // Test expectations that make sure the callbacks fire for:
+        // 1. The network response from Edge Network
+        // 2. The event response handles converted into events, captured by test case event listeners
+        let edgeRequestContentExpectation = XCTestExpectation(description: "Edge extension request content listener called")
+        let networkResponseExpectation = XCTestExpectation(description: "Network request callback called")
+        
+        // MARK: Network response assertions
+        testingDelegate.testCaseCompletion = { httpConnection in
+            print("SOURCE: testCaseCompletion: \(httpConnection)")
+            print("data as string: \(httpConnection.responseString)")
+            
+            httpConnection.response?.url
+            print("baseURL: \(httpConnection.response?.url?.host)")
+            if httpConnection.response?.url?.host == "obumobile5.data.adobedc.net" {
+                XCTAssertEqual(200, httpConnection.responseCode)
+                networkResponseExpectation.fulfill()
+            }
+            
+        }
+        
+        let validationJSON = #"""
+        {
+          "payload": [
+            {
+              "ttlSeconds" : 1800,
+              "scope" : "Target",
+              "hint" : "35"
+            },
+            {
+              "ttlSeconds" : 1800,
+              "scope" : "AAM",
+              "hint" : "9"
+            },
+            {
+              "ttlSeconds" : 1800,
+              "scope" : "EdgeNetwork",
+              "hint" : "or2"
```

Comment:
@emdobrin 
If github action is triggered from some other region, the location hint might be different right?

Reply:
@timkimadobe  Should we base expectation based on the input value for the githubAction? Or this is just a setup and will be made dynamic in the upcoming PR?

Reply:
@addb you are correct, we should capture that when we start adding test cases in future PRs. This PR should account for the GitHub action setup and making the configured params accessible to the test, this is more of an example.